proto/select.proto

```diff
@@ -91,12 +75,32 @@ message DAGRequest {
 	optional uint64 sql_mode = 9;
 
 	// It indicates whether the sql mode is strict.
-	optional bool is_strict_sql_mode = 10;
+	// Deprecated. Don't use.
+	// optional bool is_strict_sql_mode = 10;
```

Comment:
Why we need to comment it out?

Reply:
Because it is not used and is covered by `sql_mode`.